<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolGroup</class>
 <widget class="QWidget" name="ToolGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(60, 67, 83);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(255, 255, 255, 0);font: 15pt &quot;Arial&quot;;}
QPushButton:hover{background-color: rgba(255, 255, 255, 60)}</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>69</y>
     <width>64</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;可添加&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(255, 255, 255, 0);font: 20pt &quot;Arial&quot;;}
QPushButton:hover{background-color: rgba(255, 255, 255, 60)}</string>
   </property>
   <property name="text">
    <string>◉</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(255, 255, 255, 0);font: 16pt &quot;Arial&quot;;}
QPushButton:hover{background-color: rgba(255, 255, 255, 60)}</string>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
